packages/remix-adapter/src/globals.ts

```diff
@@ -0,0 +1,18 @@
+/*
+Remix provides `process.env.NODE_ENV` at compile time.
+Declare types for `process` here so that they are available in Deno.
+*/
+
+interface ProcessEnv {
+  NODE_ENV?: "development" | "production" | "test";
+  [key: string]: string | undefined;
+}
+interface Process {
+  env: ProcessEnv;
+}
+declare global {
+  // deno-lint-ignore no-var
+  var process: Process;
+}
```

Comment:
I've realised that I did exactly what happened before! We don't need to declare these types, because this is compiled for Node. We need a ts-ignore for the Deno global, but no types